src/Model.php

```diff
@@ -710,6 +710,7 @@ public function addCondition($field, $operator = null, $value = null)
 
         if ($f) {
             $f->system = true;
+            $f->ui['editable'] = false;
```

Comment:
That works, but for Reference_One fields editable=true is set as default here: https://github.com/atk4/data/blob/develop/src/Reference_One.php#L53